guides/source/getting_started.md

```diff
@@ -242,755 +241,843 @@ different _actions_. Each action's purpose is to collect information to provide
 it to a view.
 
 A view's purpose is to display this information in a human readable format. An
-important distinction to make is that it is the _controller_, not the view,
-where information is collected. The view should just display that information.
+important distinction to make is that the _controller_, not the view,
+is where information is collected. The view should just display that information.
 By default, view templates are written in a language called eRuby (Embedded
 Ruby) which is processed by the request cycle in Rails before being sent to the
 user.
 
+When we make a request to our Rails applications, we do so by making a request to a particular _route_. So to start off, we'll start with a route. Let's create one now in `config/routes.rb`:
+
+```ruby
+Rails.application.routes.draw do
+  get "/articles", to: "articles#index"
+
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+end
+```
+
+This is your application's _routing file_ which holds entries in a special
+[DSL (domain-specific language)](https://en.wikipedia.org/wiki/Domain-specific_language) that tells Rails how to connect incoming requests to
+controllers and actions.
+
+The line that we have just added says that we are going to match a `GET /welcome` request to `welcome#index`. This string passed as the `to` option represents the _controller_ and _action_ that will be responsible for handling this request.
```

Comment:
Hey @radar, thanks for the work on the guide here. I probably won't be able to review the whole thing any time soon, so I'll leave it to someone else (or I'll try to come back to it if no one picks it up eventually), but at a quick glance I noticed this should probably be `GET /articles` and `articles#index`.

Reply:
Thank you! I have fixed this one up in b482f950ed9ed6306f0fafb2c23e4414867e4d5e. 

I hope someone else can find the time to review it.